substrate-node/node/src/service.rs

```diff
@@ -123,6 +123,15 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 		})?;
 
 	if config.offchain_worker.enabled {
+		#[cfg(feature = "ocw")]
+		{
+		sp_keystore::SyncCryptoStore::sr25519_generate_new(
+			&*keystore,
+			runtime::pallet_tft_price::KEY_TYPE,
+			Some("//Alice"),
```

Comment:
Alice exists when running with `--dev` and has been created by Dylan on devnet, I don't think it is good to hardcode this account to  be the account used to update the tft price.